models/OrderPickupEventModel.ts

```diff
@@ -29,6 +30,10 @@ export class OrderPickupEventModel extends BaseEntity {
   @JoinColumn({ name: 'order' })
   orders: OrderModel[];
 
+  @OneToOne((type) => EventModel, { nullable: true })
+  @JoinColumn({ name: 'linkedEvent' })
+  linkedEvent: EventModel;
+
```

Comment:
Can you double check this relation–I found a few different syntax examples online but I believe this establishes the correct one-way relation.

Reply:
This looks good to me.